xtask/src/main.rs

```diff
@@ -40,6 +42,14 @@ pub enum Command {
 impl Xtask {
     pub fn run(&self) -> Result<()> {
         match &self.command {
+            Command::All => {
+                eprintln!("checking format and clippy...");
+                commands::Lint {}.run()?;
+                eprintln!("checking licenses...");
+                commands::Compliance {}.run()?;
+                eprintln!("running tests");
```

Comment:
The entire xtask crate use a capital letter to start a sentence.

To be honest: I don't mind! It could be all in lower case like you did. The thing is: I do mind when things are mixed: sometimes using capitals, sometimes not.

Reply:
that makes sense!